Formula/deno.rb

```diff
@@ -0,0 +1,31 @@
+class Deno < Formula
+  desc      "Secure TypeScript runtime on V8"
+  homepage  "https://github.com/denoland/deno/"
+  url       "https://github.com/denoland/deno.git",
+      :tag      => "v0.2.11",
+      :revision => "b2fb8261e700c75cfb9a626ae57bbd3098e08454"
+
+  bottle do
+  end
+
+  depends_on "ninja" => :build
+  depends_on "node" => :build
+  depends_on "openssl@1.1" => :build
+  depends_on "python@2" => :build
+  depends_on "rust" => :build
+
+  def install
+    rm_rf ["node_modules", "buildtools", "build_overrides", "testing", "tools/clang"]
```

Comment:
Can you explain why these files/directories are removed ?

Reply:
These are all symlink directories, which link deeper into a git submodule called [deno_third_party](https://github.com/denoland/deno_third_party). (No idea why they're removed here.)

Reply:
On git clone these committed symlinks are not valid symlinks (iirc they are files).
As part of tools/setup.py these symlinks are added back so that's why removing them is okay.

It may be a homebrew bug or jenkins? If you git clone and git submodule locally these symlinks will resolve correctly.

Reply:
can you try removing this.. maybe this is was fixed in https://github.com/denoland/deno/commit/bba1a0d44aa3dd58bd0de53f37982ef270c7a176